### DISCUSSION TITLE
hw1 vs hw2 for Mini Blueretro from RetroScaler

### DISCUSSION THREAD

**Discussion by mohdjameelbh** [darthcloud/BlueRetro#929] [Q&A]
I have the Mini Blueretro Receiver Adapter for PlayStation that comes with USC-C and ESP32-E chip from RetroScaler. which firmware update should I use hw1 or hw2?

for reference, the product page link is: https://aliexpress.com/item/1005006057023528.html

**Comment by darthcloud** [comment]
HW1
